### PR TITLE
update: support local file extraction

### PR DESCRIPTION
**Pull request by ZoneTwelve**
Hey bro, I really like your idea for article-md-cli. I would like to add a new feature that could extract local files.
What's changed
- Rename the url to link src/lib.ts
- import fs on src/lib.ts
- Check whether the link is a local file or not.
  - If it's a file, read the content with encoding=utf8.

I hope that you will enjoy it.

### CODE REVIEW

File: src/lib.ts [hunk 1]
```diff
@@ -35,17 +36,24 @@ ${fence}
...
+  link: string,
   turndownService: TurndownService,
   format: string,
 ): Promise<Article> {
-  const response = await axios.get(url);
-  const doc = new JSDOM(response.data);
+  let rawData = "";
+  // check the link is a local file or not
+  if(fs.existsSync(link)) {
+    // I think I going to add a encoding option: --encoding=utf8
```
Comment (aome510) [inline src/lib.ts]: saw 'utf8' as the parameter already. Can you remove the comment?

**Review by aome510 — APPROVED** [review]

File: src/lib.ts (hunk 1 above)
Comment (ZoneTwelve) [reply src/lib.ts]: I know you might not like it, but that can remind you that there could be issues in the future. 
You need to add another optional parameter for that.
And I already allow edits by maintainers.

Screenshot 2023-08-27 at 8 46 55 AM

Comment (aome510) [reply src/lib.ts]: oh so you mean the optional cli option, I thought passing 'utf8' to the function call below. Just want to make sure we're on the same page. 'utf8' by default should work fine. Can always add a cli option later if it doesn't work for someone else.

Comment (ZoneTwelve) [reply src/lib.ts]: You are right.
I mean the cli options.

**Pull request merged**